setup/pi/setup-teslausb

```diff
@@ -977,6 +977,12 @@ make_root_fs_readonly
 
 upgrade_packages
 
+if ! command -v jq &>/dev/null
```

Comment:
Since this is only used when using Tessie, this should probably go in the check_tessie_api function in the configure.sh script